vendor/bat-native-ledger/src/bat/ledger/internal/ledger_impl.cc

```diff
@@ -1106,22 +1106,29 @@ void LedgerImpl::OnReconcileCompleteSuccess(
                                                 date);
 }
 
-void LedgerImpl::RemoveRecurringTip(const std::string& publisher_key) {
-  ledger_client_->OnRemoveRecurring(
+void LedgerImpl::RemoveRecurringTip(
+    const std::string& publisher_key,
+    ledger::RemoveRecurringTipCallback callback) {
+  ledger_client_->RemoveRecurringTip(
       publisher_key,
       std::bind(&LedgerImpl::OnRemovedRecurring,
                 this,
-                _1));
+                _1,
+                callback));
 }
 
-void LedgerImpl::OnRemovedRecurring(ledger::Result result) {
+void LedgerImpl::OnRemovedRecurring(
```

Comment:
Would be good to make this `OnRemoveRecurring` for consistency with the rewards callback.

Reply:
done